frontend/javascripts/libs/UpdatableTexture.ts

```diff
@@ -134,21 +83,27 @@ class UpdatableTexture extends THREE.Texture {
     width: number,
     height: number,
   ) {
+    if (originalTexSubImage2D == null) {
+      // See explanation at declaration of originalTexSubImage2D.
+      originalTexSubImage2D = this.gl.texSubImage2D.bind(this.gl);
+      this.gl.texSubImage2D = _.noop;
+    }
```

Comment:
This workaround is a bit different from when we last talked, I think. If I see it correctly, `gl.texSubImage2D` is permanently noop'ed, instead of only temporarily during the texture initialization. Couldn't this lead to issues if the function is called somewhere else?

Reply:
It's only slightly different, because the old behavior basically also permanently noop'ed the webgl function, but temporarily swapped it back in before calling the function in `update`. After the update, the function was noop'ed again. So, if this function would be called elsewhere nothing would happen with both approaches. Therefore, I removed the temporary pointer juggling from the update function, since it didn't really have a benefit.

> instead of only temporarily during the texture initialization

This would be ideal, but I don't know how we would do this elegantly, since threejs initializes the textures at its own will (most likely on the first render call). For normal initialization, it might be feasible to override it only temporarily, but if a webgl context is lost and recovered, we would also need to plug into that (if we care about that scenario).

> Couldn't this lead to issues if the function is called somewhere else?

Yes, I think, if we would use other textures somewhere for rendering that wouldn't work anymore. However, we have never needed that which is why I think we can ignore that problem for now.

Reply:
Ah, ok makes sense, thanks for explaining. I agree that we should leave it as-is, then :)